# only look for features in this region (pixel)
roi:
    x_min: 200
    x_max: 900
    y_min: 40
    y_max: 280

# discard data when theese limits are not met
limits:
    dx_max: 0.1
    dx_min: -0.1
    dy_max: 0.1
    dy_min: -0.1
    theta_max: 0.1
    theta_min: -0.1
